atomsci/ddm/utils/model_file_reader.py

```diff
@@ -223,9 +222,9 @@ def get_model_info(self):
             model_dict['bucket'] = self.pparams.bucket
             ds_client = dsf.config_client()
             try:
-                ds_dset = ds_client.ds_datasets.get_bucket_dataset(bucket_name=bucket, dataset_key=dskey).result()
+                _ds_dset = ds_client.ds_datasets.get_bucket_dataset(bucket_name=self.pparams.bucket, dataset_key=self.pparams.dataset_key).result()
```

Comment:
This has some runtime implications, so just double check that what we suggested is correct